src/components/BifrostReactWidget.tsx

```diff
@@ -12,6 +12,11 @@ const globalStyles = (theme: any) => css`
   // Global styles for the widget
   //===========================================================
   .bifrost-widget {
+    width: calc(100vw - 200px);
```

Comment:
Since the parent container now has an enforced width, we shouldn't have to do this anymore. Maybe try a percentage of the parent? Are you going for `100%`?